components/o3-tooltip/main.css

```
@@ -0,0 +1,311 @@
+	position: relative;
+	box-sizing: border-box;
+	overflow: auto;
+	padding: var(--padding-top) var(--padding-right) var(--padding-right)
+		var(--padding-right);
+	hyphens: auto;
+}
+
+o3-tooltip-onboarding[no-title] .o3-tooltip-content {
+	margin-right: var(--icon-button-size);
+}
+
+.o3-tooltip-content-title {
+	font-weight: var(--o3-font-weight-semibold);
+	font-size: var(--o3-font-size-0);
+	color: inherit;
+	margin-bottom: var(--o3-spacing-4xs);
+}
+
+/* ARROW STYLES */
+.o3-tooltip > .o3-tooltip-wrapper > [data-tooltip-arrow],
+.o3-tooltip > .o3-tooltip-wrapper > [data-tooltip-arrow]::before {
+	--_arrow-size: 20px;
+	--_arrow-offset: calc(var(--_arrow-size) / 2);
+	--_negative-arrow-offset: calc(-1 * var(--_arrow-offset));
+
+	position: absolute;
+	width: var(--_arrow-size);
+	height: var(--_arrow-size);
+	background: inherit;
+}
+.o3-tooltip > .o3-tooltip-wrapper > [data-tooltip-arrow]::before {
+	visibility: visible;
+	content: '';
+	transform: rotate(45deg);
+}
+
+.o3-tooltip [data-tooltip-arrow] {
+	visibility: hidden;
+}
+
+[data-popper-placement^='top'] [data-tooltip-arrow] {
+	bottom: var(--_negative-arrow-offset);
+}
+
+[data-popper-placement^='bottom'] [data-tooltip-arrow] {
+	top: var(--_negative-arrow-offset);
+}
+
+[data-popper-placement^='left'] [data-tooltip-arrow] {
+	right: var(--_negative-arrow-offset);
+}
+
+[data-popper-placement^='right'] [data-tooltip-arrow] {
+	left: var(--_negative-arrow-offset);
+}
+
+[data-popper-placement='top-start'] [data-tooltip-arrow] {
+	left: var(--_arrow-offset);
+}
+
+[data-popper-placement='top-end'] [data-tooltip-arrow] {
+	right: var(--_arrow-offset);
+}
+
+[data-popper-placement='top'] [data-tooltip-arrow] {
+	left: 50%;
+	transform: translateX(-50%);
+}
+
+[data-popper-placement='bottom-start'] [data-tooltip-arrow] {
+	left: var(--_arrow-offset);
+}
+
+[data-popper-placement='bottom-end'] [data-tooltip-arrow] {
+	right: var(--_arrow-offset);
+}
+
+[data-popper-placement='bottom'] [data-tooltip-arrow] {
+	left: 50%;
+	transform: translateX(-50%);
+}
+
+[data-popper-placement='left-start'] [data-tooltip-arrow] {
+	top: var(--_arrow-offset);
+}
+
+[data-popper-placement='left-end'] [data-tooltip-arrow] {
+	bottom: var(--_arrow-offset);
+}
+
+[data-popper-placement='left'] [data-tooltip-arrow] {
+	top: 50%;
+	transform: translateY(-50%);
+}
+
+[data-popper-placement='right-start'] [data-tooltip-arrow] {
+	top: var(--_arrow-offset);
+}
+
+[data-popper-placement='right-end'] [data-tooltip-arrow] {
+	bottom: var(--_arrow-offset);
+}
+
+[data-popper-placement='right'] [data-tooltip-arrow] {
+	top: 50%;
+	transform: translateY(-50%);
+}
+
+/* Button styles */
+o3-tooltip-toggle .o3-toggletip-target,
+.o3-tooltip-wrapper .o3-tooltip-close {
+	all: unset; /* remove all default browser styles for buttons */
+	width: var(--icon-button-size);
+	height: var(--icon-button-size);
+}
+
+o3-tooltip-toggle .o3-toggletip-target::before,
+.o3-tooltip-wrapper .o3-tooltip-close::before {
```

Could these selectors align with the one below for focus states?

```suggestion
o3-tooltip-toggle button {
	all: unset; /* remove all default browser styles for buttons */
	width: var(--icon-button-size);
	height: var(--icon-button-size);
}

o3-tooltip-toggle button::before {
```